src/Ombi.Api.Jellyfin/JellyfinApi.cs

```diff
@@ -67,15 +67,15 @@ public async Task<EmbyUser> LogIn(string username, string password, string apiKe
 
             request.AddJsonBody(body);
 
-            request.AddHeader("X-Emby-Authorization",
+            request.AddHeader("X-Jellyfin-Authorization",
```

Comment:
```suggestion
            request.AddHeader("X-Emby-Authorization",
```

We don't currently have `X-Jellyfin-Authorization`

Reply:
Aah so it is still that, I wasn't 100% so just went with it.